Cargo.toml

```diff
@@ -6,17 +6,14 @@ edition = "2018"
 
 
 [dependencies]
-dusk-plonk = {version = "0.1.0"}
-dusk-bls12_381 = "0.1.0"
-hades252 = {git = "https://github.com/dusk-network/Hades252", tag = "v0.5.0"}
-poseidon252 = {git = "https://github.com/dusk-network/Poseidon252", tag = "v0.5.0"}
+dusk-plonk = "0.2.0"
+hades252 = {git = "https://github.com/dusk-network/Hades252", tag = "v0.6.0"}
+poseidon252 = {git = "https://github.com/dusk-network/Poseidon252"}
 failure = "0.1.7"
 kelvin = "0.13.0"
-jubjub = {git = "https://github.com/dusk-network/jubjub", branch = "master"}
 num-bigint = "0.2.6"
 num-traits = "0.2.11"
 
 [dev-dependencies]
-merlin = "2.0.0"
 rand = "0.7.3"
 rand_core = "0.5.1"
```

Comment:
I know is not related to this PR, but since we're editing this file could you add the new line at the end? Thanks!

Reply:
Done